sdk/webpubsub/azure-webpubsub-client/azure/webpubsub/client/_client.py

```diff
@@ -0,0 +1,864 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+from typing import Any, overload, Callable, Union, Optional, Dict, List
+import time
+import sys
+import logging
+import threading
+import urllib.parse
+import websocket  # pylint: disable=import-error
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.pipeline.policies import RetryMode
+
+from .models._models import (
+    OnConnectedArgs,
+    OnDisconnectedArgs,
+    OnServerDataMessageArgs,
+    OnGroupDataMessageArgs,
+    WebPubSubJsonProtocol,
+    WebPubSubJsonReliableProtocol,
+    SequenceId,
+    RetryPolicy,
+    WebPubSubGroup,
+    SendMessageErrorOptions,
+    WebPubSubMessage,
+    SendMessageError,
+    SendEventMessage,
+    SendToGroupMessage,
+    AckMessage,
+    ConnectedMessage,
+    SequenceAckMessage,
+    CloseEvent,
+    OnRejoinGroupFailedArgs,
+    DisconnectedMessage,
+    GroupDataMessage,
+    ServerDataMessage,
+    JoinGroupMessage,
+    LeaveGroupMessage,
+    AckMessageError,
+)
+from .models._enums import WebPubSubDataType, WebPubSubClientState, CallBackType, WebPubSubProtocolType
+from ._util import delay, format_user_agent
+
+_LOGGER = logging.getLogger(__name__)
+
+if sys.version_info >= (3, 8):
```

Comment:
You should be able to remove this check - just import from `typing_extensions` and it will do the check for you.

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/205c0e59e1cc4b8092c57a5d92fee63fdb056448/sdk/webpubsub/azure-webpubsub-client/setup.py#L68
For higher version, we don't hope install typing-extensions additionally.